MCDataProducts/inc/ProcessCode.hh

```diff
@@ -88,9 +88,9 @@ namespace mu2e {
       mu2eExternalRMC,         mu2eFlateMinus,      mu2eFlatePlus, mu2eFlatPhoton, // 175
       mu2eCePlusLeadingLog, mu2ePionCaptureAtRest, mu2eExternalRPC, mu2eInternalRPC, // 179
       mu2eCaloCalib, mu2ePienu, mu2eunused7, mu2eunused8, // 183
-      uninitialized, NoProcess, GammaGeneralProc, // 186
-      mu2eGammaConversion, Radioactivation, nCaptureHP, nFissionHP, mu2eAntiproton, // 191
-      mu2eFlatMuMinus, mu2eFlatMuPlus, // 193
+      uninitialized, NoProcess, GammaGeneralProc, mu2eGammaConversion, // 187
+      Radioactivation, nCaptureHP, nFissionHP, mu2eAntiproton, // 191
+      mu2eFlatMuMinus, mu2eFlatMuPlus, CoupledTransportation, // 194
```

Comment:
This block of codes typically is for mu2e physics processes. What is CoupledTransportation? Is it mu2e specific?
Instead of extending the process code list please overwrite one of the existing ' unused' or mu2eunused' codes.

Reply:
CoupleTransportation is used for tracking in parallel (scoring) world, it is a generic Geant4 code. I'll admit I didn't read the full list of codes, I'll try to find an unused slot (ah, California's obsession with recycling....)